src/components/kronisk/KroniskSide.tsx

```diff
@@ -154,13 +154,12 @@ const KroniskSide = () => {
             <Systemtittel>Arbeidssituasjon og miljø</Systemtittel>
             <br />
             <SkjemaGruppe>
-              <Normaltekst>Vi spør først om dere har forsøkt å løse situasjonen på arbeidsplassen.</Normaltekst>
-              <Normaltekst>Svaret deres brukes i to forskjellige vurderinger:</Normaltekst>
+              <Normaltekst>
+                Vi spør først om arbeidsdagen og hvordan den påvirker den ansatte. Svaret deres brukes til å vurdere om
```

Comment:
Det hadde vært fint om du oversatte siden i samme slengen. Se på BekreftOpplysningerPanel.tsx for et eksempel på hvordan det kan gjøres. Vi oversetter til engelsk selv.